daemon/cmd/status.go

```diff
@@ -599,16 +600,22 @@ func (d *Daemon) getStatus(brief bool) models.StatusResponse {
 
 	sr.Stale = stale
 
+	//CiliumVersion definition
+	Ver := v.GetCiliumVersion()
+	ciliumVer := fmt.Sprintf("    Ver: %s  Rev: %s", Ver.Version, Ver.Revision)
+
 	switch {
 	case len(sr.Stale) > 0:
+		msg := "Stale status data"
 		sr.Cilium = &models.Status{
 			State: models.StatusStateWarning,
-			Msg:   "Stale status data",
+			Msg:   msg + ciliumVer,
```

Comment:
while string concatenation is faster, it's hard to read and error prone (e.g. ciliumVer _must_ have space prefix, msg should be ending with dot or comma, etc). I would prefer to go with fmt.Sprintf() in general.

Reply:
> Thanks for PR.
> 
> Based on my understanding, version will be printed in case of any failed check. This could be
> redudant in some cases. What do you think about always printing version details at the beginning ?

Thanks Tam for the suggestion! Printing version details at the beginning is a good idea. But there are some entities which printing status at the beginning. For example:
$ cilium status
KVStore:                Ok   etcd: 1/1 connected, lease-ID=29c6767063b65989, lock lease-ID=29c6767063b6598b, has-quorum=true: https://192.168.33.11:2379 - 3.4.13 (Leader)
Kubernetes:             Ok   1.20+ (v1.20.0-rc.0) [linux/amd64]
Cilium:                 Ok            Ver: 1.9.90  Rev: 25d8615be
Cilium health daemon:   Ok   
BandwidthManager:       Disabled
Host Routing:           Legacy
Proxy Status:           OK, ip 10.11.0.128, 0 redirects active on ports 10000-20000
Hubble:                 Ok              Current/Max Flows: 471/4095 (11.50%), Flows/s: 11.86   Metrics: Disabled

Maybe we will return to this idea, but a little later, after feedback? Because perhaps it would be more correct to make the output uniform for all entities.

Reply:
> while string concatenation is faster, it's hard to read and error prone (e.g. ciliumVer _must_ have space prefix, msg should be ending with dot or comma, etc). I would prefer to go with fmt.Sprintf() in general.

OK. I've implemented fmt.Sprintf() in the next commit. And regarding the question about the order of output I swapped msg and ciliumVer. Could you please check:
$ cilium status
Cilium:                 Failure   [Ver: 1.9.90  Rev: 25d8615be]    Kvstore service is not ready